specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2021-12-01-preview/Tables.json

```diff
@@ -291,6 +291,48 @@
           "final-state-via": "azure-async-operation"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/tables/{tableName}/migrate": {
+      "post": {
+        "tags": [
+          "Tables"
+        ],
+        "x-ms-examples": {
+          "TablesGet": {
+            "$ref": "./examples/TablesMigrate.json"
+          }
+        },
+        "operationId": "Tables_Migrate",
+        "description": "Migrate a Log Analytics workspace table.",
```

Comment:
this description should indicate what it is migrating the table to.  The API doesn't take input and it is not obvious

Reply:
Hi @pilor, thank you for the comment. a better description was provided.

Reply:
Signing off but you may want to consider taking a body with an enum indicating what migration is being performed.  This action has a very generic name "migrate" and will be confusing if you have another migration scenario in the future